recipes/coiled-cloud/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "coiled-cloud" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bfd51fcaf67e5cf209e1982977fe2a1b5f69e22d79f6a01217c8b2dcc5a72b1b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - aiohttp
+    - aiobotocore >=0.12.0,<0.13
+    - distributed
+
+test:
```

You should add `pip check` to ensure that the requirements constraints match the Python package. This will prevent issues in building the package with that tight `aiobotocore` pin.

```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```